src/app/pages/AddDestination/AddDestination.tsx

```diff
@@ -2,39 +2,36 @@ import React from 'react';
 import { useNavigate } from 'react-router';
 import styled from 'styled-components';
 import useLocalStorage from '../../hooks/useLocalStorage';
+import type { DestinationType } from '../../utils/DestinationType';
 
 export default function AddDestination(): JSX.Element {
-  const destination = {
+  const newDestination: DestinationType = {
     location: '',
     startDate: '',
     endDate: '',
   };
   const navigate = useNavigate();
 
-  const [newLocation, setNewLocation] = useLocalStorage<string>(
+  const [newLocation, setNewLocation] = useLocalStorage(
     'location',
-    destination.location
+    newDestination.location
   );
-  const [startDate, setStartDate] = useLocalStorage<string>(
+  const [startDate, setStartDate] = useLocalStorage(
     'startDate',
-    destination.startDate
+    newDestination.startDate
   );
-  const [endDate, setEndDate] = useLocalStorage<string>(
+  const [endDate, setEndDate] = useLocalStorage(
     'endDate',
-    destination.endDate
+    newDestination.startDate
```

Comment:
shoud be     newDestination.`end`Date, right?

Reply:
Yes! Copy/Paste error 🙄  Thanks for spotting!